db/schema.rb

```diff
@@ -36,12 +36,31 @@
     t.index ["quiz_id"], name: "index_questions_on_quiz_id", using: :btree
   end
 
+  create_table "quizmasters", force: :cascade do |t|
```

Comment:
How did you create this table? I don't see any migration file for this... Do we actually need this?